### PR TITLE
Chatbot Guardrails Arena (release 21 March if possible)

### PR DESCRIPTION
**Pull request by clefourrier**
Mostly for a light review: is it understandable, are there obvious questions the reader would ask themselves?

### CODE REVIEW

File: arena-lighthouz.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
+---
+title: "Introducing the Chatbot Guardrails Arena"
+thumbnail: /blog/assets/arenas-on-the-hub/thumbnail_lighthouz.png
+authors:
+- user: Sonali Pattnaik
```
Comment (clefourrier) [inline arena-lighthouz.md:5]: This needs to be completed, waiting for the info

File: arena-lighthouz.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+Recognizing this, Lighthouz AI has launched the Chatbot Guardrails Arena, in collaboration with HuggingFace, to stress test LLMs and privacy guardrails in leaking sensitive data. 
+
+Put on your creative caps! Chat with two anonymous LLMs with guardrails and try to trick them into revealing sensitive financial information. Cast your vote for the model that demonstrates greater privacy. The votes will be compiled into a leaderboard showcasing the LLMs and guardrails rated highest by the community for their privacy.
+
+Our vision behind the Chatbot Guardrails Arena is to establish the trusted benchmark for AI chatbot security, privacy, and guardrails. With a large-scale blind stress test by end users, this arena will offer an unbiased and practical assessment of the reliability of current privacy guardrails. 
+
+<script type="module" src="https://gradio.s3-us-west-2.amazonaws.com/3.45.1/gradio.js"> </script>
```
Comment (osanseviero) [inline arena-lighthouz.md:26]: It should match https://huggingface.co/spaces/lighthouzai/guardrails-arena/blob/main/README.md

```suggestion
<script type="module" src="https://gradio.s3-us-west-2.amazonaws.com/4.21.0/gradio.js"> </script>
```

File: arena-lighthouz.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+<gradio-app theme_mode="light" space="lighthouzai/guardrails-arena"></gradio-app>
+
+
+## Why Stress Test Privacy Guardrails?
+
+Data privacy is crucial even if you are building an internal-facing AI chatbot/agent – imagine one employee being able to trick an internal chatbot to find out another employee’s SSN, home address, or salary information. The need for data privacy is obvious when building external-facing AI chatbots/agents – you don’t want customers to have unauthorised access to company information.
```
Comment (osanseviero) [inline arena-lighthouz.md:32]:
```suggestion
Data privacy is crucial even if you are building an internal-facing AI chatbot/agent – imagine one employee being able to trick an internal chatbot into finding another employee’s SSN, home address, or salary information. The need for data privacy is obvious when building external-facing AI chatbots/agents – you don’t want customers to have unauthorised access to company information.
```

File: arena-lighthouz.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+
+<script type="module" src="https://gradio.s3-us-west-2.amazonaws.com/3.45.1/gradio.js"> </script>
+<gradio-app theme_mode="light" space="lighthouzai/guardrails-arena"></gradio-app>
+
+
+## Why Stress Test Privacy Guardrails?
+
+Data privacy is crucial even if you are building an internal-facing AI chatbot/agent – imagine one employee being able to trick an internal chatbot to find out another employee’s SSN, home address, or salary information. The need for data privacy is obvious when building external-facing AI chatbots/agents – you don’t want customers to have unauthorised access to company information. 
+
+Currently, there is no systematic study evaluating the privacy of AI chatbots, as far as we are aware. This arena bridges this gap with an initial focus on privacy of AI chatbots, however, we expect the learnings to inform the development of privacy-preserving AI agents and AI assistants in the future as well.
```
Comment (osanseviero) [inline arena-lighthouz.md:34]:
```suggestion
Currently, there is no systematic study evaluating the privacy of AI chatbots, as far as we are aware. This arena bridges this gap with an initial focus on the privacy of AI chatbots. However, we expect the learnings to inform the development of privacy-preserving AI agents and AI assistants in the future as well.
```

File: arena-lighthouz.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+
+## The Leaderboard
+
+The votes collected from the Guardrails Arena will be compiled and displayed on a dedicated public leaderboard. At the moment, the leaderboard is empty, but it will start to fill with privacy rankings of all 12 LLMs with guardrails once a substantial number of votes have been collected. As more votes are submitted, the leaderboard will be updated in real-time, reflecting the ongoing assessment of model safety. 
+
+As is accepted practice, similar to [LMSYS](https://lmsys.org/)'s [Chatbot Arena](https://huggingface.co/spaces/lmsys/chatbot-arena-leaderboard) & the community’s [TTS arena and leaderboard](https://huggingface.co/blog/arena-tts), the ranking will be based on the [Elo rating system](https://en.wikipedia.org/wiki/Elo_rating_system).
+
+## How is the Chatbot Guardrails Arena different from other Chatbot Arenas?
+
+Traditional chatbot arenas, like the [LMSYS chatbot arena](https://arena.lmsys.org/), aim to measure the overall conversational quality of LLMs. The participants in these arenas converse on any general topic and rate based on their own judgement of response “quality”.
```
Comment (osanseviero) [inline arena-lighthouz.md:65]:
```suggestion
Traditional chatbot arenas, like the [LMSYS chatbot arena](https://arena.lmsys.org/), aim to measure the overall conversational quality of LLMs. The participants in these arenas converse on any general topic and rate based on their judgment of response “quality”. 
```

File: arena-lighthouz.md [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+
+Traditional chatbot arenas, like the [LMSYS chatbot arena](https://arena.lmsys.org/), aim to measure the overall conversational quality of LLMs. The participants in these arenas converse on any general topic and rate based on their own judgement of response “quality”. 
+
+On the other hand, in the Chatbot Guardrails Arena, the goal is to measure LLMs and guardrails' data privacy capabilities. To do so, the participant needs to act adversarially to extract secret information known to the chatbots. Participants vote based on the capability of preserving the secret information. 
+
+## Taking Part in the Next Steps
+
+The Chatbot Guardrails Arena kickstarts the community stress testing of AI applications’ privacy concerns. By contributing to this platform, you’re not only stress testing the limits of AI and the current guardrail system but you’re actively participating in the definition of its ethical boundaries. Whether you’re a developer, an AI enthusiast, or simply curious about the future of technology, your participation matters. Participate in the arena, cast your vote, and share your successes with others on social media!
```
Comment (osanseviero) [inline arena-lighthouz.md:71]:
```suggestion
The Chatbot Guardrails Arena kickstarts the community stress testing of AI applications’ privacy concerns. By contributing to this platform, you’re not only stress-testing the limits of AI and the current guardrail system but actively participating in defining its ethical boundaries. Whether you’re a developer, an AI enthusiast, or simply curious about the future of technology, your participation matters. Participate in the arena, cast your vote, and share your successes with others on social media! 
```

**Review by osanseviero — APPROVED** [review]
Nice!

File: arena-lighthouz.md [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+- user: Sonali Pattnaik
+  guest: true
+- user: Rohan Karan
+  guest: true
+- user: Srijan Kumar
+  guest: true
+- user: clefourrier
+---
+
+# Introducing the Chatbot Guardrails Arena
```
Comment (osanseviero) [inline arena-lighthouz.md:14]: I feel the first few paragraphs of the blog post have a strong "sales" kinda pitch. I would suggest to strongly reduce the tone here to have a stronger focus in the leaderboard directly

File: arena-lighthouz.md [hunk 8]
```diff
@@ -0,0 +1,75 @@
...
+- user: rohankaran
+  guest: true
+- user: srijankedia
+  guest: true
+- user: clefourrier
+---
+
+# Introducing the Chatbot Guardrails Arena
+
+With the recent jumps in augmented LLM capabilities, deployment of enterprise AI assistants (such as chatbots and agents) with access to internal databases is likely to increase; this trend could help with many tasks, from internal document summarization to personalised customer support. However, data privacy of said databases can be a serious concern (see [1](https://www.forrester.com/report/security-and-privacy-concerns-are-the-biggest-barriers-to-adopting/RES180179), [2](https://retool.com/reports/state-of-ai-2023) and [3](https://www.mckinsey.com/capabilities/quantumblack/our-insights/the-state-of-ai-in-2023-generative-ais-breakout-year#/)) when deploying these models in production. So far, guardrails have emerged as the widely accepted technique to ensure the quality, security, and privacy of AI chatbots, but [anecdotal evidence](https://incidentdatabase.ai/) suggests that even the best guardrails can be circumvented with relative ease.
```
Comment (lighthouzai) [inline arena-lighthouz.md:16]: Thanks for the review, minor changes: "With the recent advancements in augmented LLM capabilities, deployment of enterprise AI assistants (such as chatbots and agents) with access to internal databases is likely to increase; this trend could help with many tasks, from internal document summarization to personalized customer and employee support. However, data privacy of said databases can be a serious concern (see [1](https://www.forrester.com/report/security-and-privacy-concerns-are-the-biggest-barriers-to-adopting/RES180179), [2](https://retool.com/reports/state-of-ai-2023) and [3](https://www.mckinsey.com/capabilities/quantumblack/our-insights/the-state-of-ai-in-2023-generative-ais-breakout-year#/)) when deploying these models in production. So far, guardrails have emerged as the widely accepted technique to ensure the quality, security, and privacy of AI chatbots, but [anecdotal evidence](https://incidentdatabase.ai/) suggests that even the best guardrails can be circumvented with relative ease. "

Comment (clefourrier) [inline arena-lighthouz.md:16]:
```suggestion
With the recent advancements in augmented LLM capabilities, deployment of enterprise AI assistants (such as chatbots and agents) with access to internal databases is likely to increase; this trend could help with many tasks, from internal document summarization to personalized customer and employee support. However, data privacy of said databases can be a serious concern (see [1](https://www.forrester.com/report/security-and-privacy-concerns-are-the-biggest-barriers-to-adopting/RES180179), [2](https://retool.com/reports/state-of-ai-2023) and [3](https://www.mckinsey.com/capabilities/quantumblack/our-insights/the-state-of-ai-in-2023-generative-ais-breakout-year#/)) when deploying these models in production. So far, guardrails have emerged as the widely accepted technique to ensure the quality, security, and privacy of AI chatbots, but [anecdotal evidence](https://incidentdatabase.ai/) suggests that even the best guardrails can be circumvented with relative ease.
```

File: arena-lighthouz.md [hunk 9]
```diff
@@ -0,0 +1,75 @@
...
+
+# Introducing the Chatbot Guardrails Arena
+
+With the recent advancements in augmented LLM capabilities, deployment of enterprise AI assistants (such as chatbots and agents) with access to internal databases is likely to increase; this trend could help with many tasks, from internal document summarization to personalized customer and employee support. However, data privacy of said databases can be a serious concern (see [1](https://www.forrester.com/report/security-and-privacy-concerns-are-the-biggest-barriers-to-adopting/RES180179), [2](https://retool.com/reports/state-of-ai-2023) and [3](https://www.mckinsey.com/capabilities/quantumblack/our-insights/the-state-of-ai-in-2023-generative-ais-breakout-year#/)) when deploying these models in production. So far, guardrails have emerged as the widely accepted technique to ensure the quality, security, and privacy of AI chatbots, but [anecdotal evidence](https://incidentdatabase.ai/) suggests that even the best guardrails can be circumvented with relative ease.
+
+Lighthouz AI is therefore launching the Chatbot Guardrails Arena in collaboration with Hugging Face, to stress test LLMs and privacy guardrails in leaking sensitive data.
```
Comment (clefourrier) [inline arena-lighthouz.md:18]:
```suggestion
[Lighthouz AI](https://lighthouz.ai/) is therefore launching the Chatbot Guardrails Arena in collaboration with Hugging Face, to stress test LLMs and privacy guardrails in leaking sensitive data.
```

**Pull request merged**